rca/enquire_to_study/views.py

```diff
@@ -33,6 +36,32 @@ def post_qs(self, form_data):
     def create_form_submission(self, form):
         form.save()
 
+    def send_user_email_notification(self, form):
+        enquiry_form_settings = EnquireToStudySettings.objects.first()
+        if not enquiry_form_settings.email_submission_notifations:
+            return
+        email_content = enquiry_form_settings.email_content
+        # add links to programmes for the response
+        if "programmes" in form.cleaned_data:
+            email_content += (
+                "\n\nPlease see the following pages for more "
+                "information about the courses you enquired about: \n"
+            )
+            for programme in form.cleaned_data["programmes"]:
+                email_content += (
+                    f"{programme.title} {settings.BASE_URL + programme.url} \n"
+                )
```

Comment:
AC: As a user, if I have enquired about a programme, then my email notification includes a link to the relevant programme page(s) for the programme(s) i have selected